GMG/Scenes/Export/ExportIntent.swift

```diff
@@ -15,29 +15,34 @@ final class ExportIntent: ExportIntentProtocol {
         self.model = model
     }
 
-    func onAppear() {
-        model.readScore { score in
-            let renderer = ImageRenderer(content: ChordSheetView(score: score))
-
-            if let uiImage = renderer.uiImage {
-                model.updateSheetImage(uiImage)
-
-                if let data = uiImage.pngData() {
-                    let fileName = "sheet-\(score.id.uuidString).png"
-                    let url = FileManager.default
-                        .temporaryDirectory
-                        .appendingPathComponent(fileName)
-
-                    do {
-                        try data.write(to: url)
-                        model.updateSheetURL(url)
-                    } catch {
-                        print("error")
-                    }
+    func onAppear(_ score: Score) {
+
+        let renderer = ImageRenderer(content: ChordSheetView(score: score))
+
+        if let uiImage = renderer.uiImage {
+            model.updateSheetImage(uiImage)
+
+            if let data = uiImage.pngData() {
+                let fileName = "sheet-\(score.id.uuidString).png"
+                let url = FileManager.default
+                    .temporaryDirectory
+                    .appendingPathComponent(fileName)
+
+                do {
+                    try data.write(to: url)
+                    model.updateSheetURL(url)
+                } catch {
+                    print("error")
                 }
             }
+        }
 
-            model.updateAudioURL(score.audioURL)
+        do {
+            let renderedURL = try DefaultScoreAudioRenderer(score: score)
+                .renderToAudioFile(score: score, fileName: "\(score.title).m4a")
+            model.updateAudioURL(renderedURL)
+        } catch {
+            print("offline render failed: \(error)")
         }
```

Comment:
Renderer 이니셜라이즈랑 분리하는 것이 어떨까욥

```suggestion
        do {
            let renderer: ScoreAudioRenderer = DefaultScoreAudioRenderer(score: score)
            let renderedURL: URL = try renderer.renderToAudioFile(fileName: "\(score.title).m4a")
            model.updateAudioURL(renderedURL)
        } catch {
            print("offline render failed: \(error)")
}
```